generic_tracers/generic_tracer.F90

```diff
@@ -687,8 +687,8 @@ subroutine generic_tracer_vertdiff_M(dh, dhw, diff_cbt, dt, Rho_0,tau)
 
     !nnz: Should I loop here or inside the sub g_tracer_vertdiff ?    
     !JGJ 2013/05/31  merged COBALT into siena_201303
-    if(do_generic_age .or. do_generic_argon .or. do_generic_CFC .or. do_generic_TOPAZ .or. do_generic_ERGOM &
-       .or. do_generic_BLING .or. do_generic_miniBLING .or. do_generic_COBALT .or. do_generic_blres) then
+    if(do_generic_age .or. do_generic_argon .or. do_generic_CFC &
+       .or. do_generic_BLING .or. do_generic_COBALT .or. do_generic_blres) then
```

Comment:
Also `if(do_generic_tracer) then` here? But, abiotic and SF6 are missing here. Can we actually delete generic_tracer_vertdiff_M (in a separate PR)?

Reply:
#48

Reply:
@andrew-c-ross, good point. Personally, I would love to remove unused modules or subroutines like `generic_tracer_vertdiff_M`, but I would like to hear from other people as well. Perhaps we can discuss this this afternoon.